catalog.html

```diff
@@ -0,0 +1,268 @@
+          </li>
+          </ul>
+
+      </nav>
+
+    </header>
+    <main class="main-inner">
+      <header class="inner-header">
+        <h1 class="inner-header-title">Гостиницы Седоны</h1>
+        <ul class="breadcrumbs">
+          <li class="breadcrumbs-item">
+            <a class="visually-hidden" href="index.html">Главная</a>
+          </li>
+          <li class="breadcrumbs-item">
+            <a class="breadcrumbs-link">Гостиницы</a>
+          </li>
+        </ul>
+        <section class="catalog-products">
+          <h2 class="visually-hidden">Инфраструктура и тип жилья</h2>
+          <form class="catalog-filter" action="https://echo.htmlacademy.ru/" method="get">
+            <fieldset class="catalog-filter-group">
+              <legend class="catalog-filter-title">Инфраструктура</legend>
+
+              <label class="catalog-filter-item">
+                <input class="control-input" type="checkbox" name="swimming-pool" checked>
+
+                Бассейн
+              </label>
+
+              <label class="catalog-filter-item">
+                <input class="control-input" type="checkbox" name="parking-space" checked>
+
+                Парковка
+              </label>
+
+              <label class="catalog-filter-item">
+                <input class="control-input" type="checkbox" name="wi-fi" >
+
+                Wi-Fi
+              </label>
+
+            </fieldset>
+
+            <fieldset class="catalog-filter-group">
+            <legend class="catalog-filter-title">Тип жилья</legend>
+            <ul class="catalog-filter-list">
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="house-groups" value="hotel" checked>
+
+              Гостиница
+                </label>
+              </li>
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="house-groups" value="Motel" >
+
+              Мотель
+                </label>
+              </li>
+              <li class="catalog-filter-item">
+                <label class="control">
+                  <input class="control-input" type="radio" name="house-groups" value="apartments">
+
+              Апартаменты
+                </label>
+              </li>
+            </ul>
+
+          </fieldset>
+          <fieldset>
+            <legend class="catalog-filter-group"> Стоимость, ₽:</legend>
+            от
+            <input class="conteol-input-min" type="number" name="min-price" value="0">
+            До
+            <input class="conteol-input-max" type="number" name="min-price" value="9000">
+            <input class="cjntrol-input" type="range" name="price-range">
+          </fieldset>
+
+            <button class="button" type="submit">Применить</button>
+            <button class="button" type="submit">Сбросить</button>
```

Comment:
```suggestion
            <button class="button" type="reset">Сбросить</button>
```
Это кнопка отмены (очистка полей формы).